manifests/0000_20_kube-apiserver-operator_08_flowschema.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
+kind: PriorityLevelConfiguration
+metadata:
+  name: openshift-system
```

Comment:
This should be allowed more requests than `openshift-control-plane-operators`.  Even though it's one server, really don't want to run out.

Also, rename `aggregated-api-delegated-auth` I think.  We'll want to tie the oauth-apiserver to it too, right?

Reply:
> Also, rename aggregated-api-delegated-auth
sounds good, I am adding a prefix `openshift` to it. 

> We'll want to tie the oauth-apiserver to it too, right?
I have added oauth apiserver. 

> This should be allowed more requests than openshift-control-plane-operators. Even though it's one server, really don't want to run out.
Yes, makes sense. we expect the operators to have a steady call rate (unless there is a bug that causes a hot loop or something) and openshift api servers are tied to the user traffic. I doubled the concurrency share to 20 and reduced for the operators to 10.